testsuite/features/build_validation/init_clients/slemicro54_ssh_minion.feature

```diff
@@ -0,0 +1,41 @@
+# Copyright (c) 2023 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+# Beware: After altering the system e.g. package installation/removal, the system
+#         has to be rebooted to take effect due to the nature of SLE Micro.
+
+@slemicro54_ssh_minion
+Feature: Bootstrap a SLE Micro 5.4 Salt SSH minion
+
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
+
+  Scenario: Bootstrap a SLE Micro 5.4 SSH minion
+    When I follow the left menu "Systems > Bootstrapping"
+    Then I should see a "Bootstrap Minions" text
+    When I check "manageWithSSH"
+    And I enter the hostname of "slemicro54_ssh_minion" as "hostname"
+    And I enter "linux" as "password"
```

Comment:
Are we missing user and port here?

Reply:
No by default it takes root if you don't specify anything and port is by default 22